WebService/appsettings.ini

```diff
@@ -44,7 +44,7 @@ twin_read_write_enabled = "${?PCS_TWIN_READ_WRITE_ENABLED}"
 # Connection string for the Azure storage account used to create and delete devices in bulk
 # Format: DefaultEndpointsProtocol=https;AccountName=_____;AccountKey=_____;EndpointSuffix=core.windows.net
 iothub_import_storage_account_connstring = "${PCS_AZURE_STORAGE_ACCOUNT}"
-
+user_agent = "devicesimulation"
```

Comment:
Please add a comment explaining what this setting does and why it exists.

Reply:
done